### ISSUE TITLE
Layout of autocomplete field broken in changelist

### ISSUE THREAD

**Issue by raimondi1337** [unfoldadmin/django-unfold#326]
```
@admin.register(Game)
class GameAdmin(ModelAdmin):
    list_display_links = ["title"]
    list_display = ["title", "studio", "rating"]
    list_editable = ["studio", "rating"]
```
    
    
<img width="1449" alt="Screenshot 2024-03-19 at 4 52 31 PM" src="https://github.com/unfoldadmin/django-unfold/assets/518383/6cbbbdd5-138c-4e45-8d16-ba37d131f5fa">

**Comment by raimondi1337** [comment]
> +1, this would be a great boon for us if it worked correctly

I fixed it for myself with this
`./myapp/static/admin.css`
```
.related-widget-wrapper > select {
  width: auto;
  background-color: transparent;
}

.related-widget-wrapper-link {
  display: none !important;
}
```

`./myproject.settings.py`
```
UNFOLD = {
    ...,
    "STYLES": [
        lambda request: static("admin.css"),
    ],
}
```

**Comment by lukasvinclav** [comment]
Thanks for report. Fixed in #335
